_migration-assistant/deploying-migration-assistant/configuration-options.md

```diff
@@ -170,6 +170,46 @@ The `serviceSigningName` can be `es` for an Elasticsearch or OpenSearch domain,
 
 All of these authentication options apply to both source and target clusters.
 
+## Bring your own snapshot options
+
+An existing snapshot can alternatively be used to perform metadata as well as backfill migrations, instead of using the Migration Assistant to create a snapshot.
```

Comment:
```suggestion
An existing Amazon S3 snapshot can alternatively be used to perform metadata as well as backfill migrations, instead of using the Migration Assistant to create a snapshot.
```